chebi-update/src/main/java/org/reactome/release/chebiupdate/ChebiUpdater.java

```diff
@@ -115,76 +152,112 @@ public void updateChebiReferenceMolecules() throws SQLException, Exception
 			String chebiName = entity.getChebiAsciiName();
 			List<DataItem> chebiFormulae = entity.getFormulae();
 
-			updateReferenceEntities(molecule, chebiName, instanceEdit);
 
-			// Now, check to see if we need to update the ReferenceMolecule itself.
 			String moleculeIdentifier = (String) molecule.getAttributeValue(ReactomeJavaConstants.identifier);
 			String moleculeName = (String) molecule.getAttributeValuesList(ReactomeJavaConstants.name).get(0);
 			String moleculeFormulae = (String) molecule.getAttributeValue(ReactomeJavaConstants.formula);
 
```

Comment:
Could these three Strings be moved into the private methods that use them?  That way you could also get rid of them as parameters and use the 'molecule' parameter to get their values inside the methods.